source/fe/mapping_fe.cc

```diff
@@ -909,12 +909,82 @@ MappingFE<dim, spacedim>::transform_real_to_unit_cell(
   const typename Triangulation<dim, spacedim>::cell_iterator &cell,
   const Point<spacedim> &                                     p) const
 {
-  Assert(false, StandardExceptions::ExcNotImplemented());
+  const std::vector<Point<spacedim>> support_points =
+    this->compute_mapping_support_points(cell);
+
+  Tensor<1, spacedim> DF[dim];
+  // [TODO]
+  // When 2nd derivatives are implemented, we'll uncomment this
+  // Tensor<1, spacedim> D2F[dim][dim];
```

Comment:
Why not do it right away?

Reply:
2nd derivatives are not implemented for Simplex::FE_P (yet)

Reply:
I am on it!